conda-forge.yml

```
@@ -13,3 +13,4 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+test: native_and_emulated
```

I thought we have QEMU for osx-arm64, but maybe I misremembered. Set this and rerender:
```suggestion
test: native
```
See https://conda-forge.org/docs/maintainer/conda_forge_yml/#test.

---

Hmm, but for osx arm64 is it emulated or just cross-compiled without an emulator? It looks like in the last build it skipped the test stage. In the osx arm64 cross-compile docs it suggests `native_and_emulated`. I wonder what would have changed?

https://conda-forge.org/blog/2020/10/29/macos-arm64/#cross-compiling-builds-for-osx-arm64

---

Ah OK, so it was just a matter of forgetting to rerender. Yes it makes sense now.

conda-forge has the very nice capability that one can cross-compile on the bare metal, but at the test stage spinning up QEMU to run actual tests. I think this is what `native_and_emulated` offers. I was just puzzled that why it's not applicable to osx-arm64, without checking about re-render. Nice gotach, Will. Thanks.